src/renderer/components/pool/PoolDetails.helpers.ts

```diff
@@ -33,14 +45,53 @@ export const getMembers = (data: Pick<PoolStatsDetail, 'uniqueMemberCount'>) =>
 export const getFees = (data: Pick<PoolStatsDetail, 'totalFees'>, priceRatio: BigNumber = bn(1)) =>
   baseToAsset(baseAmount(bnOrZero(data.totalFees).multipliedBy(priceRatio)))
 
-export const getEarnings = (
-  oData: O.Option<Pick<EarningsHistoryItemPool, 'earnings'>>,
+export const getILPPaid = (
+  { impermanentLossProtectionPaid }: Pick<PoolStatsDetail, 'impermanentLossProtectionPaid'>,
   priceRatio: BigNumber = bn(1)
-) =>
-  FP.pipe(
-    oData,
-    O.fold(
-      () => ZERO_ASSET_AMOUNT,
-      (data) => baseToAsset(baseAmount(bnOrZero(data.earnings).multipliedBy(priceRatio)))
-    )
-  )
+) => baseToAsset(baseAmount(bnOrZero(impermanentLossProtectionPaid).multipliedBy(priceRatio)))
+
+export const getEmptyPoolDetail = (): PoolDetail => ({
+  asset: 'asset',
+  assetDepth: '0',
+  assetPrice: '0',
+  assetPriceUSD: '0',
+  poolAPY: '0',
+  runeDepth: '0',
+  status: '',
+  units: '0',
+  volume24h: ''
+})
+
+export const getEmptyPoolStatsDetail = (): PoolStatsDetail => ({
+  addAssetLiquidityVolume: '0',
+  addLiquidityCount: '0',
+  addLiquidityVolume: '0',
+  addRuneLiquidityVolume: '0',
+  asset: 'asset',
+  assetDepth: '0',
+  assetPrice: '0',
+  assetPriceUSD: '0',
+  averageSlip: '0',
+  poolAPY: '0',
+  runeDepth: '0',
+  status: 'Staged',
+  swapCount: '0',
+  swapVolume: '0',
+  toAssetAverageSlip: '0',
+  toAssetCount: '0',
+  toAssetFees: '0',
+  toAssetVolume: '0',
+  toRuneAverageSlip: '0',
+  toRuneCount: '0',
+  toRuneFees: '0',
+  toRuneVolume: '0',
+  totalFees: '0',
+  uniqueMemberCount: '0',
+  uniqueSwapperCount: '0',
+  units: '0',
+  withdrawAssetVolume: '0',
+  withdrawCount: '0',
+  withdrawRuneVolume: '0',
+  withdrawVolume: '0',
+  impermanentLossProtectionPaid: '0'
+})
```

Comment:
maybe there is a sense to place it at the `src/renderer/helpers/poolHelper.ts` ?

Reply:
^ These functions are used by `PoolDetails` only (not by other places / components). That's why they have been moved into  `PoolDetails.helpers.ts` `

Reply:
i have no problems with placing it here just for the future possible cases it might be needed at another places (maybe, maybe no). lets keep it here but also keep in mind that we have such empty-data generator for `poolDetail` and `poolStatsDetail`